Starting job at Sun Feb 16 06:13:17 PM CET 2025
Running Filter_by_part.py for subdirectory 22010, part 11
2025-02-16 18:13:17 - INFO - Event filtering process starts...
2025-02-16 18:13:17 - INFO - Initialized PureNeutrinoEventFilter filter for 22010/11
2025-02-16 18:13:18 - INFO - Loaded 10 event specifier files from /lustre/hpc/project/icecube/HE_Nu_Aske_Oct2024/clean_events_dict/22010/22010/reduced
2025-02-16 18:13:21 - INFO - Selected 0 CSV files covering event range [5205, 6749]
2025-02-16 18:13:21 - WARNING - No matching CSV files found for 22010/11
2025-02-16 18:13:21 - INFO - Loaded 0 valid (RunID, EventID) pairs from CSV files
2025-02-16 18:13:27 - WARNING - No valid events in 22010/11. Skipping filtering.
2025-02-16 18:13:27 - INFO - Initialized Muon13Filter filter for 22010/11
2025-02-16 18:13:30 - INFO - Extracted 2493068 valid events where MuonFilter_13==True.
2025-02-16 18:13:30 - ERROR - An error occurred: 'PureNeutrinoEventFilter' object has no attribute 'valid_event_nos'
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 65, in <module>
    main()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 61, in main
    run()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 48, in run
    filter_manager = EventFilterManager(
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 19, in __init__
    self.filter_obj = self._build_Filter_object()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 54, in _build_Filter_object
    valid_event_nos = self._synthesize_valid_event_nos()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 78, in _synthesize_valid_event_nos
    event_sets = {alias: f.get_valid_event_nos() for alias, f in self.filter_instances.items()}
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 78, in <dictcomp>
    event_sets = {alias: f.get_valid_event_nos() for alias, f in self.filter_instances.items()}
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilter.py", line 36, in get_valid_event_nos
    return self.valid_event_nos
AttributeError: 'PureNeutrinoEventFilter' object has no attribute 'valid_event_nos'
Job completed at Sun Feb 16 06:13:30 PM CET 2025
